### ISSUE TITLE
Encrypted or corrupted DB file on 3.1.+

### ISSUE THREAD

**Issue by noelmansour** [realm/realm-java#4462]
#### Goal

An unencrypted DB.

#### Expected Results

Viewing the Realm DB in the mac browser without an encryption key.

#### Actual Results

The mac browser asks for an encryption key for the Realm DB. Though, it could be a red herring and the DB file is corrupted.

#### Steps & Code to Reproduce

Create a new project that uses realm-java 3.1.0 or 3.1.1. Pull the file and open it in the Mac browser:

`adb shell run-as com.test.myapplication cat /data/data/com.test.myapplication/files/default.realm > default.realm && open default.realm
`

I also pulled the file using `adb pull` on a rooted device and the same issue occurred.

I don't think this an issue with the Mac browser because it doesn't happen on realm-java 3.0.0.

#### Code Sample

Any project using realm will do but this is a very minimal project that you can use to repro the issue: https://github.com/noelmansour/realm-corrupt-file

#### Version of Realm and tooling
Realm version(s): 3.1.0, 3.1.1

Realm sync feature enabled: no

Android Studio version: 2.3.1

Which Android version and device: Nexus 5 running 7.1.1, Nexus 5X running 7.1.1

**Comment by bmeike** [comment]
Have you tried using `adb pull`.  My first guess is that `cat` is corrupting the db.

**Comment by nicholas-jomo** [comment]
Did you update your Mac Realm browser? That's what fixed the issue for me a while back ago.

**Comment by noelmansour** [comment]
@nicholas-jomo Yup, that was the issue. The weird thing is that my Realm Browser indicated that it was version 2.1.5. When I checked the app store, it said I was up-to-date, but I could also see that 2.1.7 was the latest version. So I deleted the app and reinstalled. Then when I checked, it was version 2.1.7.

I guess it was an issue with the Mac Browser 😅 

I'll close this and open an issue with the Mac Browser project.
